composer.lock

```diff
@@ -2887,69 +2887,6 @@
                 "source": "https://git.drupalcode.org/project/autosave_form"
             }
         },
-        {
-            "name": "drupal/back_to_top",
```

Comment:
Not sure why it's removing the back to top module from composer.lock, It's listed in composer.json, so think this needs to be added back in.